.github/workflows/beaker.yml

```diff
@@ -151,6 +151,11 @@ jobs:
       - name: install additional packages
         if: ${{ inputs.additional_packages != '' }}
         run: sudo apt-get install -y ${{ inputs.additional_packages }}
+      - name: Setup libvirt for Vagrant
+        if: ${{ inputs.beaker_hypervisor == 'vagrant' }}
```

Comment:
As mentioned in https://github.com/voxpupuli/puppet-mosquitto/pull/50/files#r1461846550, I think this should be `vagrant_libvirt`. `vagrant` is AFAIK Vagrant with VirtualBox.